README.md

```diff
@@ -158,6 +158,7 @@ The above command pulled the existing data out of Route53 and placed the results
 | [Ns1Provider](/octodns/provider/ns1.py) | All | No | |
 | [OVH](/octodns/provider/ovh.py) | A, AAAA, CNAME, MX, NAPTR, NS, PTR, SPF, SRV, SSHFP, TXT | No | |
 | [PowerDnsProvider](/octodns/provider/powerdns.py) | All | No | |
+| [Rackspace](/octodns/provider/rackspace.py) | A, AAAA, ALIAS, CNAME, MX, NS, PTR, SPF, TXT | No |  |
```

Comment:
This cover everything they support or a partial list based on what the provider has built out?

Reply:
This is just what we are confident we support fully. RackSpace supports more record types than this, however, we found enough idiosyncrasies in these that we didn't want to try adding any record types we didn't have extensive testing for. It should be easy to add more support in future if someone has a need and enough good test data to check their assumptions.